pkgs/tools/admin/exoscale-cli/default.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, buildGo114Package, fetchFromGitHub }:
+
+buildGo114Package {
```

Needed to use `version = "v${version}"` in `fetchFromGitHub`.


```suggestion
buildGo114Package rec {
```